lib/parallel_tests/pids.rb

```diff
@@ -0,0 +1,68 @@
+require 'tempfile'
+require 'singleton'
+require 'json'
+
+module ParallelTests
+  class Pids
+    attr_reader :file, :pids, :mutex
```

Comment:
mutex/file/pids should not be public ?

Reply:
@grosser I'll have time to clean up and address these issues this weekend. Thanks!